zerver/tests/test_migrations.py

```diff
@@ -344,8 +344,8 @@ def test_multiple_stream_and_topic_edits(self) -> None:
         )
 
 
-class MessageEditAfterMigrationWithEditHistory(MigrationsTestCase):
-    __unittest_skip__ = False
+class MessageEditAfterMigrationWithEditHistory(MigrationsTestCase):  # nocoverage
+    __unittest_skip__ = True
```

Comment:
Should we just delete these tests in this commit; we don't necessarily keep any but the example and there's little point in keeping tests we won't run?